internal/nginx/maxmind.go

```diff
@@ -60,15 +73,53 @@ func GeoLite2DBExists() bool {
 
 // DownloadGeoLite2DB downloads the required databases by the
 // GeoIP2 NGINX module using a license key from MaxMind.
-func DownloadGeoLite2DB() error {
-	for _, dbName := range strings.Split(MaxmindEditionIDs, ",") {
-		err := downloadDatabase(dbName)
-		if err != nil {
-			return err
-		}
-		MaxmindEditionFiles = append(MaxmindEditionFiles, dbName+dbExtension)
+func DownloadGeoLite2DB(period time.Duration, attempts int) error {
```

Comment:
nit:  the order of these two parameters should be reversed, IMO.

Reply:
Fixed